### PR TITLE
Updating CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by clouderati**
Cleaning up CONTRIBUTING.md file a bit:

* removing ^M newlines
* removing references to VMware

Signed-off-by: clouderati <35942204+clouderati@users.noreply.github.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,303 +1,303 @@
...
+│   ├── security
+│   └── utils
+├── jobservice          # Source code for the job service component
+│   ├── api
+│   ├── config
+│   ├── job
+│   ├── replication
+│   ├── scan
+│   └── utils
+├── ui                  # Source code for the harbor service component
```
Comment (steven-zou) [inline CONTRIBUTING.md]: "ui" is moved to "core" already

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,303 +1,303 @@
...
+
+#### Go
+Harbor backend is written in [Go](http://golang.org/). If you don't have a Harbor backend service development environment, please [set one up](https://golang.org/doc/install).
+
+|  Harbor  |  Requires Go  |
+|----------|---------------|
+|   1.1    |    1.7.3      |
+|   1.2    |    1.7.3      |
+|   1.3    |    1.9.2      |
+|   1.4    |    1.9.2      |
```
Comment (steven-zou) [inline CONTRIBUTING.md]: How about appending the latest two other versions 1.5, 1.6 here?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,303 +1,303 @@
...
+#### Web
+
+Harbor web UI is built based on [Clarity](https://vmware.github.io/clarity/) and [Angular](https://angular.io/) web framework. To setup web UI development environment, please make sure the [npm](https://www.npmjs.com/get-npm) tool is installed first.
+
+|  Harbor  |  Requires Angular  |  Requires Clarity  |
+|----------|--------------------|--------------------|
+|   1.1    |      2.4.1         |       0.8.7        |
+|   1.2    |      4.1.3         |       0.9.8        |
+|   1.3    |      4.3.0         |       0.10.17      |
+|   1.4    |                    |                    |
```
Comment (steven-zou) [inline CONTRIBUTING.md]: Same comments with above

File: CONTRIBUTING.md (hunk 2 above)
Comment (clouderati) [reply CONTRIBUTING.md]: Do you happen to know what versions of Go were required for these two versions?

File: CONTRIBUTING.md (hunk 3 above)
Comment (clouderati) [reply CONTRIBUTING.md]: Ditto above with Angular / Clarity @zhoumeina

**Comment by clouderati** [comment]
@steven-zou and @zhoumeina can you ptal at the two questions I shared above re: supported Go, Angular / Clarity versions? Thanks!

File: CONTRIBUTING.md (hunk 2 above)
Comment (steven-zou) [reply CONTRIBUTING.md]: @clouderati Same with 1.3&1.4, still keep at 1.9.2

File: CONTRIBUTING.md (hunk 3 above)
Comment (steven-zou) [reply CONTRIBUTING.md]: @ninjadq 

Please help to provide the required info for @clouderati .

Comment (ninjadq) [reply CONTRIBUTING.md]: | 1.4 | 4.3.0 | 0.10.17 |
| 1.5 | 4.3.0 | 0.10.27 |
| 1.6 | 4.3.0 | 0.10.27 |

**Comment by clouderati** [comment]
Just resolved a few comments. Can you ptal @steven-zou? Thanks!

**Comment by clouderati** [comment]
Rebased + resolved conflicts

**Comment by clouderati** [comment]
`push -f` to fix merge conflict

**Pull request closed without merging**